src/Server.cpp

```diff
@@ -9,6 +9,19 @@ Server::~Server()
 {
 }
 
+/* Work in progress, I'm sure we're going to need more stuff here, but this 
+* 	should let us spin up a server with an explicit call. --Dan */
```

Comment:
Not sure this comment is necessary, once one leaves comments that need to cleaned up after wards, they rarely actually are :D rather just put this on the PR and we can find it on version control!

Reply:
done